dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/exception/UnableToForkException.java

```diff
@@ -0,0 +1,14 @@
+package com.salesforce.dockerfileimageupdate.exception;
+
+
+/**
+ * Indicates that there was an error while trying to fork a Git repository.
+ */
+public class UnableToForkException extends Exception {
```

Comment:
This seems unnecessary. Does the GitHub library not send a typed exception for this?

Reply:
Github library always throws IOException. I thought it would be more readable this way.

Reply:
Github library always throws IOException.